integration-tests/src/test/java/oracle/weblogic/kubernetes/ItJrfMiiDomain.java

```diff
@@ -0,0 +1,293 @@
+    logger.info("Create RCU access secret");
+    String rcuaccessSecretName = domainUid + "-rcu-access";
+    logger.info("Creating RCU access secret: {0}, with prefix: {1}, dbUrl: {2}, schemapassword: {3})",
+        rcuaccessSecretName, RCUSCHEMAPREFIX, RCUSCHEMAPASSWORD, dbUrl);
+    assertDoesNotThrow(() -> createRcuAccessSecret(
+        rcuaccessSecretName,
+        jrfDomainNamespace,
+        RCUSCHEMAPREFIX,
+        RCUSCHEMAPASSWORD,
+        dbUrl),
+        String.format("createSecret failed for %s", rcuaccessSecretName));
+
+    logger.info("Create OPSS wallet password secret");
+    String opsswalletpassSecretName = domainUid + "-opss-wallet-password-secret";
+    assertDoesNotThrow(() -> createOpsswalletpasswordSecret(
+        opsswalletpassSecretName,
+        jrfDomainNamespace,
+        "welcome1"),
+        String.format("createSecret failed for %s", opsswalletpassSecretName));
+
+    logger.info("Create an image with jrf model file");
+
+    // build the model file list
+    final List<String> modelList = Collections.singletonList(MODEL_DIR + "/" + modelFile);
+    jrfMiiImage = createMiiImageAndVerify(
+        "jrf-mii-image",
+        modelList,
+        Collections.singletonList(MII_BASIC_APP_NAME),
+        FMWINFRA_IMAGE_NAME,
+        FMWINFRA_IMAGE_TAG,
+        "JRF",
+        false);
+
+    // push the image to a registry to make it accessible in multi-node cluster
+    dockerLoginAndPushImageToRegistry(jrfMiiImage);
+
+    // create the domain object
+    Domain domain = createDomainResource(domainUid,
+        jrfDomainNamespace,
+        adminSecretName,
+        OCIR_SECRET_NAME,
+        encryptionSecretName,
+        rcuaccessSecretName,
+        opsswalletpassSecretName,
+        replicaCount,
+        jrfMiiImage);
+
+    // create model in image domain
+    createDomainAndVerify(domain, jrfDomainNamespace);
+
+    //check admin service created
+    logger.info("Check admin service {0} is created in namespace {1}",
+        adminServerPodName, jrfDomainNamespace);
+    checkServiceExists(adminServerPodName, jrfDomainNamespace);
+
+    // check admin server pod is ready
+    logger.info("Wait for admin server pod {0} to be ready in namespace {1}",
+        adminServerPodName, jrfDomainNamespace);
+    checkPodReady(adminServerPodName, domainUid, jrfDomainNamespace);
+
+    // check managed server services created
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Check managed service {0} is created in namespace {1}",
+          managedServerPrefix + i + "-c1", jrfDomainNamespace);
+      checkServiceExists(managedServerPrefix + i + "-c1", jrfDomainNamespace);
+    }
+
+    // check managed server pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Wait for managed pod {0} to be ready in namespace {1}",
+          managedServerPrefix + i + "-c1", jrfDomainNamespace);
+      checkPodReady(managedServerPrefix + i + "-c1", domainUid, jrfDomainNamespace);
+    }
```

Comment:
Verify few JRF scenarios.  Check accessibility of default JRF application or check em console.